kratos/modeler/modeler_factory.h

```diff
@@ -0,0 +1,57 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+
+
+#if !defined(KRATOS_MODELER_FACTORY_H_INCLUDED )
+#define  KRATOS_MODELER_FACTORY_H_INCLUDED
+
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/kratos_components.h"
+#include "modeler/modeler.h"
+
+namespace Kratos
+{
+///@name Kratos Classes
+///@{
+
+/// Creates Modelers.
+namespace ModelerFactory
```

Comment:
this should be merged with #3185 (separate PR)

Reply:
Ok but we can do this as soon as this PR is merged?